docker-machine-nfs.sh

```diff
@@ -429,8 +429,16 @@ isNFSMounted()
 {
   for shared_folder in "${prop_shared_folders[@]}"
   do
+    case $prop_machine_driver in
+      vmwarefusion)
+        nfs_mount_name="vmhgfs-fuse on $shared_folder"
```

Comment:
Are you sure about this part? I downloaded a trial of vmwarefusion and when creation the machine **_wihout running docker-machine-nfs_** it already shows "vmhgfs-fuse on /Users type fuse.vmhgfs-fuse". 
This means that when you run docker-machine-nfs it will actually not do anything because the isNFSMounted() function will say that NFS is already configured while it is not.

Reply:
I'll check later today. I think that means we'd just need to check /etc/exports to make sure NFS was configured properly.